### DISCUSSION TITLE
How to return specific state using getters?

### DISCUSSION THREAD

**Discussion by iamshulz** [vuejs/pinia#700] [Help and Questions] [answered]
Is there a way for me to return multiple state on getters?

I know I can just call all the values of the state from exporting `defineStore`. I prefer to have specific state return upon calling the getter. 

Most of the examples show only one state being returned from a getter.

```
state: () => ({
    name: '', //get this
    sku: '', //get this
    department: '', //get this
    description: '',
  }),
```

**Comment by posva** [comment, marked as answer]
You can return an object:

```js
defineStore('id', {
  state: () => ({ ... }),
  getters: {
    stuff: state => ({ one: state.one, two, state.two }),
  },
})
```

Remember you can directly use the state of the store and that there is [mapState()](https://pinia.esm.dev/api/modules/pinia.html#mapstate) and [storeToRefs()](https://pinia.esm.dev/core-concepts/#using-the-store) to help

**Reply by iamshulz** [reply]
Thank you ! I was missing the property name!
